src/quo2/components/notifications/activity_logs.cljs

```diff
@@ -171,32 +189,28 @@
            replying?
            unread?]
     :as   props}]
-  [rn/view {:flex-direction     :row
-            :align-items        :flex-start
-            :border-radius      16
-            :padding-top        8
-            :padding-horizontal (if replying? 20 12)
-            :padding-bottom     12
-            :background-color   (when (and unread? (not replying?))
-                                  colors/primary-50-opa-10)}
+  [rn/view {:accessibility-label :activity
```

Comment:
should these be in a `style.cljs` file? 🤔

Reply:
They should, but initially I opened this PR only for fixing bugs. Brian also suggested I make this refactoring, so I'm going to quickly do it given the quorum ;)